src/java.base/share/classes/java/util/Timer.java

```diff
@@ -322,6 +322,11 @@ public void schedule(TimerTask task, Date firstTime, long period) {
      * scheduling multiple repeating timer tasks that must remain synchronized
      * with respect to one another.
      *
+     * @crac Since the execution is suspended after a checkpoint the timer
+     * could execute many times after a restore. This is likely an undesired
+     * behaviour, therefore it is recommended to cancel the task before
+     * checkpoint and schedule it again after restore.
```

Comment:
In this wording this looks like a problem, and I'm not convinced that the problem exists (otherwise we may want probably to do something about that).

Could you change the description to something more neutral? "could execute many times after a restore, catching up as described above. If this is not desirable,  the task can be canceled and scheduled again in a  {@link Resource} implementation"